### ISSUE TITLE
[20230618] Weekly AI ArXiv 만담 시즌2 - 20회차

### ISSUE THREAD

**Issue by jungwoo-ha** [jungwoo-ha/WeeklyArxivTalk#86]
### ArXiv

**Comment by veritas9872** [comment]
# News

## AMD에서 GPU 시장에 진출하기 위해 최근 노력이 활발해지고 있습니다.

https://www.amd.com/en/newsroom/press-releases/2023-6-13-amd-expands-leadership-data-center-portfolio-with-.html
https://huggingface.co/blog/huggingface-and-amd

현재 NVIDIA CUDA의 독점 체제에서 AMD가 가장 유사한 시장 위치에 있으나 CUDA 등 여러 장벽에 의해 영향을 가지지 못했습니다. 그러나 PyTorch에서도 AMD의 ROCm backend를 지원하며 이번에 HuggingFace에서도 AMD를 도입하기 위해 적극적으로 노력하고 있어 Python frontend를 사용하는 대다수의 유저는 곧 코드를 그대로 사용하면서 AMD를 사용할 수 있는 방향으로 움직임이 있습니다.

## Artificial Artificial Artificial Intelligence: Crowd Workers Widely Use Large Language Models for Text Production Tasks
ArXiv: https://arxiv.org/abs/2306.07899
GitHub: https://github.com/epfl-dlab/GPTurk

<img width="899" alt="image" src="https://github.com/jungwoo-ha/WeeklyArxivTalk/assets/33523965/aa590c06-8a40-402b-be7f-98027b22b7b0">

<img width="511" alt="image" src="https://github.com/jungwoo-ha/WeeklyArxivTalk/assets/33523965/aeeba50d-33e7-4c95-a99a-b23e013e7c1f">

**Comment by veritas9872** [comment]
# Research

## Tree-Ring Watermarks: Fingerprints for Diffusion Images that are Invisible and Robust
ArXiv: https://arxiv.org/abs/2305.20030
GitHub: https://github.com/YuxinWenRick/tree-ring-watermark

![image](https://github.com/jungwoo-ha/WeeklyArxivTalk/assets/33523965/cf7fb622-9c69-4748-8042-5c9162dff605)

**Comment by nick-jhlee** [comment]
## News
- [Evidence for the use of quantum computing before fault tolerance](https://www.nature.com/articles/s41586-023-06096-3)
  - demonstrated for the first time that quantum computers with 100+ qubits can produce accurate results - and reach beyond leading classical approaches
  - new error mitigation!
![image](https://github.com/jungwoo-ha/WeeklyArxivTalk/assets/38721329/68a598f5-5bff-456d-b267-31d032ce35b8)


## Papers
- [Simplicity Bias Leads to Amplified Performance Disparities](https://arxiv.org/abs/2212.06641) (FAccT 2023)
  - 배경: SGD has simplicity bias (bias towards a simple solution, i.e., prioritize learning majority class)
  - How to quantify this? **difficulty disparity, difficulty amplification factor**
  - This is *model-specific*! (even with a balanced dataset)
  - Conclusion: we should use a model-specific fairness audit (post-training audit)
<img width="1085" alt="Screenshot 2023-06-18 at 7 57 12 PM" src="https://github.com/jungwoo-ha/WeeklyArxivTalk/assets/38721329/fbdaa271-009d-495b-bf06-0c9a687ca051">

- "Controversial" paper: [Exploring the MIT Mathematics and EECS Curriculum Using Large Language Models](https://arxiv.org/abs/2306.08997) (MIT, Boston University)
  - GPT4가 MIT EECS를 100% 맞는다........?
  - https://flower-nutria-41d.notion.site/No-GPT4-can-t-ace-MIT-b27e6796ab5a48368127a98216c76864
  - TL;DR:
    - **Evaluation**: The authors' evaluation uses GPT 4 to score itself and repeatedly prompts until the correct answer is reached.
    - **Dataset leakage**: Significant leakage and duplication in the prompts.
  - 아직 멀었는걸까요,, ㅎㅎ

-  Recent papers in optimization theory:
  - new optimizers
     - [Sophia: A Scalable Stochastic Second-order Optimizer for Language Model Pre-training](https://arxiv.org/abs/2305.14342) (arXiv 2023)
  - **parameter-free** SGD! (no need to tune initial learning rate)
     - [Making SGD Parameter-Free](https://proceedings.mlr.press/v178/carmon22a.html) (COLT 2022)
     - [Prodigy: An Expeditiously Adaptive Parameter-Free Learner](https://arxiv.org/abs/2306.06101) (arXiv 2023)
  - ...etc

- Explosion in NLP Theories!! (수학에 관심있으신 분들은 한번 ~~내년부터~~ 같이 해보시죠 ㅎㅎ)
  - 최적화 관점에서!
    - [A Kernel-Based View of Language Model Fine-Tuning](https://arxiv.org/abs/2210.05643) (arXiv 2023)
  - Hidden mechanism of how ICL (in-context learning) works
    - ex) [Transformers as Statisticians: Provable In-Context Learning with In-Context Algorithm Selection](https://arxiv.org/abs/2306.04637)

**Comment by sujungleeml** [comment]
Health system-scale language models are all-purpose prediction engines[https://www.nature.com/articles/s41586-023-06160-y] 

- 

![image](https://github.com/jungwoo-ha/WeeklyArxivTalk/assets/56566861/d88d814e-dc43-4232-a148-938b06cb7397)
